runestone/assess/js/timed.js

```diff
@@ -486,6 +503,21 @@ Timed.prototype.tookTimedExam = function () {
 };
 
 Timed.prototype.finishAssessment = function () {
+<<<<<<< HEAD
```

Comment:
Hi - I've resolved the conflicts and would like to try again.


Thomas Hubbard




Date: Sun, 11 Oct 2015 14:17:29 -0700
From: notifications@github.com
To: RunestoneComponents@noreply.github.com
CC: thomashubbard@hotmail.com
Subject: Re: [RunestoneComponents] Update from original (#98)

In runestone/assess/js/timed.js:

> @@ -486,6 +503,21 @@ Timed.prototype.tookTimedExam = function () {
>  };
>  
>  Timed.prototype.finishAssessment = function () {
> +<<<<<<< HEAD

You  did not resolve the conflicts fully.  see here and line 520 and 534


—
Reply to this email directly or view it on GitHub.